### PR TITLE
[SC-1728] [SC-1729] [SC-1732]

### PR DESCRIPTION
**Pull request by zyuzka**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/SC-1729

###### Change log

- added function for stopping and removing all running CLI containers
---
#### Overview

- Ticket: https://spryker.atlassian.net/browse/SC-1728

###### Change log

- added new container for testing
- updated configuration for api tests
- refactored install recipe
- refactored function for building endpoint map
---

#### Overview

- Ticket: https://spryker.atlassian.net/browse/SC-1732

###### Change log
Introduced `docker/sdk wait` functionality to check availability of requested services, e.g database, broker, etc.

### CODE REVIEW

**Comment by zyuzka** [comment]
https://github.com/spryker/docker-sdk/pull/14
https://github.com/spryker/docker-sdk/pull/13
https://github.com/spryker/docker-sdk/pull/12

File: generator/src/templates/docker-compose.test.yml.twig [hunk 1]
```diff
@@ -1,10 +1,36 @@
 version: "3.5"
 
+x-volumes:
```
Comment (sprymiker) [inline generator/src/templates/docker-compose.test.yml.twig]: This block is the same as in docker-compose.yml.twig. Please move it to separate twig and include everywhere it is needed

File: generator/src/templates/deploy.bash.twig [hunk 2]
```diff
@@ -639,80 +671,47 @@ case $1 in
...
-            esac
+    wait)
+        if [ "$#" == 0 ];
+        then
+            showWaitErrorMessage
+            exit 1
+        fi
+
+        verbose "${INFO}Checking services...${NC}"
+        for services in "${@}"
```
Comment (sprymiker) [inline generator/src/templates/deploy.bash.twig]: `for service`

File: generator/src/templates/docker-compose.test.yml.twig (hunk 1 above)
Comment (sprymiker) [reply generator/src/templates/docker-compose.test.yml.twig]: application-volumes.yml.twig

**Pull request merged**